internal/pkg/service/buffer/api/receive/quota/quota.go

```diff
@@ -6,24 +6,36 @@ package quota
 import (
 	"context"
 	"sync"
+	"time"
 
 	"github.com/benbjohnson/clock"
 	"github.com/c2h5oh/datasize"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/config"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/statistics"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	commonErrors "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
-type Quota struct {
+const (
+	// MinErrorLogInterval defines the minimum interval between logged quota errors, per receiver and API node.
+	// It prevents repeating errors from flooding the log.
+	MinErrorLogInterval = 5 * time.Minute
```

Comment:
Tato konstanta urcuje, ako bude obmedzene logovanie.
- Je to per receiver a API node.
- Tj. ak mame 3 API nodes napojene na load balancer, tak sa chyba zaloguje 3x, a potom 5 min nic.